de.tudarmstadt.ukp.wikipedia.wikimachine/src/main/java/de/tudarmstadt/ukp/wikipedia/wikimachine/debug/MailMemoryLogger.java

```diff
@@ -29,29 +30,34 @@
 import javax.mail.internet.InternetAddress;
 import javax.mail.internet.MimeMessage;
 
-import org.apache.log4j.Logger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-public class MailMemorylLogger extends AbstractLogger {
+/**
+ * A logger implementation which directs its logging output an SMTP mail server endpoint.
+ *
+ * @deprecated Don't use this in production code/environments as there are open runtime issues with the SMTP config.
+ */
+@Deprecated
+public class MailMemoryLogger extends AbstractLogger {
 
-	private static final Logger log4j = Logger
-			.getLogger(MailMemorylLogger.class);
+	private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
 	private static final String DATE_FORMAT_NOW = "yyyy.MM.dd HH:mm:ss";
 
+	// TODO Do these hard-coded values make sense here?
 	private static final String ADDRESS_TO = "mail.logger.system@googlemail.com";
-
 	private static final String ADDRESS_FROM = "i_galkin@rbg.informatik.tu-darmstadt.de";
 
 	/**
-	 * @see {@link http
-	 *      ://gnu.gds.tuwien.ac.at/software/classpathx/javamail/javadoc /gnu/
-	 *      mail/providers/smtp/package-summary.html}
+	 * For configuration reference:
+	 * @see <a href="https://docs.oracle.com/javaee/7/api/javax/mail/package-summary.html>JavaMail SMTP</a>
 	 */
-
 	private static final Properties TRANSPORT_PROPERTIES = new Properties() {
 		private static final long serialVersionUID = 1L;
 
 		{
+			// TODO externalize these properties to an external properties file that is bootstrapped via different Maven profiles
 			this.put("mail.transport.protocol", "smtp");
```

Comment:
Any idea why GitHub highlights this text deep red? Maybe some spurious invisible unicode character that should be removed?

Reply:
I got absolutely no idea. I re-typed the content and there is no change from the SCM tracking's perspective, even when typed in an external plain text editor... character encoding seems to be fine. 

Maybe, this is a protection feature to highlight "configuration" of these protocols/servers, so dev won't expose there internal config to the public?